torchgeo/trainers/utils.py

```diff
@@ -11,6 +11,24 @@
 import torch.nn as nn
 from torch import Tensor
 from torch.nn.modules import Conv2d, Module
+from torchvision.models.detection.transform import GeneralizedRCNNTransform
+
+
+class GeneralizedRCNNTransformNoOp(GeneralizedRCNNTransform):  # type: ignore[misc]
+    """GeneralizedRCNNTransform without the normalize and resize ops.
+
+    .. versionadded:: 0.7.1
```

Comment:
```suggestion
    .. versionadded:: 0.8
```
It won't appear in the docs anyway, so let's pretend it is added in 0.8 even though it will be there in 0.7.1.